src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -199,6 +204,37 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         this.fetchFeedItems();
     };
 
+    handleActiveAnnotation = (feedItems: FeedItems): void => {
+        const { emitAnnotatorActiveChangeEvent, file, match, onVersionChange } = this.props;
+        const { annotatorState: { activeAnnotationId } = {} } = this.state;
+        const { annotationId } = match.params;
+        const { fileVersionId } = match.params;
+        const currentFileVersionId = file.file_version.id;
+
+        // If annotatorState does not have an activeAnnotationId but the router path has an annotationId,
+        // then we are dealing with a hard load of a deeply linked annotation
+        if (activeAnnotationId || !annotationId) {
+            return;
+        }
+
+        if (currentFileVersionId === fileVersionId) {
+            // If the pathname matched file version is the current version of the file, attempt to set
+            // the active annotation in box-annotations to the specified annotationId
+            emitAnnotatorActiveChangeEvent(annotationId);
```

Comment:
Seems like we should emit this immediately rather than waiting on the feed items fetch to finish. We should know if the version is current or not much sooner. It doesn't really seem to fit here.